runtime/testy/Ansible.py

```diff
@@ -0,0 +1,98 @@
+import socket
+import threading
+import time
+import fake_dawn
+import random
+import sys
+from runtimeUtil import *
+data = [0] #for testing purposes
+sendPort = 1235
+recvPort = 1236
+
+#Custom buffer for handling states. Holds two states, updates one and sends the other.
+class two_buffer(): 
+    def __init__(self):
+        self.data = [0, 0]
+        self.put_index = 0
+        self.get_index = 1
+    def replace(self, item):
+        self.data[self.put_index] = item
+        self.put_index = (self.put_index + 1) % 2
+        self.get_index = (self.get_index + 1) % 2
+    def get(self):
+        return self.data[self.get_index]
+
+sendBuffer = two_buffer()
+recvBuffer = two_buffer()
+raw_fake_data = [[0]]
```

Comment:
This variable is not used anywhere. (It does not look like an export to outside of this module, and there is no `global raw_fake_data` declaration anywhere)